big-dipper/imports/ui/pages/AcitvityTable.jsx

```diff
@@ -114,15 +118,20 @@ function AcitvityTable({}) {
                   <div className="user-profile">
                     {getMedia(
                       item.itemFormat,
-                      item.itemImg ? item.itemImg : item.item_img
```

Comment:
Code is good, but I just saw that Function Name's misspelled as "AcitivityTable" instead of "ActivityTable". Perhaps we need a separate PR to fix that.

Reply:
Spelling mistake fixed for ActivityTable.
PR Updated

Reply:
@Kamran95 sounds good.